src/app.ts

```diff
@@ -168,6 +169,10 @@ function addCustomCSS(windowElement: BrowserWindow): void {
   }
 }
 
+function openExternalUrl(url: string): void {
+  shell.openExternal(url)
+}
+
```

Comment:
😂 

You understood me wrong.

I meant you should wrap L313-L337 into a function, so everywhere where an external URL can be opened, should have the same behavior with trusting URLs.